src/drivers/MemoryDriver.ts

```diff
@@ -0,0 +1,57 @@
+import { IDriver } from "./IDriver";
+
+type Table = Map<string, any>;
+
+export class MemoryDriver implements IDriver {
+    public $store = new Map<string, Table>();
```

Comment:
I don't see the point in using the dollar sign as a variable prefix.
You change it to only `store`